src/routes/auth/allowance.tsx

```diff
@@ -17,24 +16,22 @@ import {
 } from "@arconnect/components";
 import Wrapper from "~components/auth/Wrapper";
 import browser from "webextension-polyfill";
-import Head from "~components/popup/Head";
 import App from "~components/auth/App";
 import Arweave from "arweave";
 import styled from "styled-components";
 import { defaultGateway } from "~gateways/gateway";
 import BigNumber from "bignumber.js";
+import { useCurrentAuthRequest } from "~utils/auth/auth.hooks";
+import { HeadAuth } from "~components/HeadAuth";
+import { AuthButtons } from "~components/auth/AuthButtons";
 
 export default function Allowance() {
   const arweave = new Arweave(defaultGateway);
 
-  // connect params
-  const params = useAuthParams<{
```

Comment:
`useCurrentAuthRequest()` replaces both `useAUthParams` and `useAuthUtils` in all Auth popup screens.

Reply:
👍